Project_source/app/src/main/java/android/clicker/school_live_simulator/ComputerCourseState.kt

```diff
@@ -0,0 +1,35 @@
+package android.clicker.school_live_simulator
+
+abstract class ComputerCourseState: StudyCourseState() {
+
+    /**
+     * This is a price of computer course
```

Comment:
I think it is better to remove this is in every commentary. Just _price of computer_ , _list of smth_

Reply:
Fixed